CHANGELOG.md

```diff
@@ -4,22 +4,77 @@ All notable changes to this project will be documented in this file.  See
 [standard-version](https://github.com/conventional-changelog/standard-version)
 for commit guidelines.
 
+## Not yet released
+
+### Features and enhancements
+
+* [ide] Most of the error messages appearing on the snackbar made persistent,
+  that is they don’t hide automatically. (#1284)
+* [ide] Better reject invalid literals in the inspector for byte values. (#1313)
+* [core] Now a character surrounded by single quotes (`'G'`) is a valid byte
+  literal which denotes a corresponding value from the ASCII table. (#1317)
+* [core] Make some error messages more clear and consistent (#1330).
+* [ide] Add View menu items to toggle the project browser and inspector (#1340).
+* [ide] Allow links between inputs and outputs of the same node. It will
+  transpile successfully if the node contains a `defer` on the loop path inside
```

Comment:
Isn't "inside it" is excessive?
IMHO, it's much better to place `defer` node outside node, cause sometimes it could be evaluated once.
So we don't need to defer results on the next transaction.

E.G.
Sometimes I want to read just one byte and work with the result. 
Sometimes I want to read byte by byte, so I'll place `defer` and link it back to `READ` pin.

Reply:
> IMHO, it's much better to place defer node outside node, cause sometimes it could be evaluated once.

Maybe. You’re talking about best practices. And I declare changes done. What works best is arguable, but technically linking an output to input is no longer an error as it was before 0.22.0.